helpers/jwt-service.go

```diff
@@ -0,0 +1,44 @@
+package helpers
+
+import (
+	"os"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/joho/godotenv"
+)
+
+var (
+	jwtServiceCreds JWTServiceCreds = NewJWTServiceCreds()
+)
+
+type JWTServiceCreds struct {
+	secretKey string
+	issuer    string
+}
+
+func NewJWTServiceCreds() JWTServiceCreds {
+	envError := godotenv.Load()
```

Comment:
I don't think you need to load it again since it was already loaded in the main func